### PR TITLE
Update README.md template

### PR DESCRIPTION
**Pull request by rhaning**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,28 +1,63 @@
...
 
-# Governance
+## 
+**Objective**
 
-The [CHARTER.md](CHARTER.md) outlines the scope and governance of our group activities.
+[What is to be achieved with this initiative]
 
-## Communications
+[OKRs]
```
Comment (dlorenc) [inline README.md:17]: OKRs is the only thing here I'd rather leave out. I've never seen them work well in an open source context, where everyone has varying/random time commitments and interesting.

Comment (kimsterv) [inline README.md:17]: Do you have examples of other OSS efforts that have done OKRs successfully? To me, OKRs seem like a stretch for open source projects. I don't think I'd be successful assigning work to people outside my company.

File: README.md [hunk 2]
```diff
@@ -1,28 +1,63 @@
...
+# 
+**Governance**
+
+
+The [CHARTER.md](https://github.com/ossf/project-template/blob/main/CHARTER.md) outlines the scope and governance of our group activities.
+
+
+
+*   Lead name 
+*   Co-Lead name
```
Comment (dlorenc) [inline README.md:63]: We're also going to need to be more flexible here long-term. For example, any WGs with funding will likely need to be a lot more detailed than those without.

Comment (rhaning) [reply README.md:63]: Agreed. We can update this when the details of funding have been finalized by the GB.

File: README.md (hunk 1 above)
Comment (rhaning) [reply README.md:17]: I view it more as a means for clearly defining the goals of the WG, not to hold specific people accountable for work. I would consider these optional as well.

Comment (rhaning) [reply README.md:17]: see my comments to Kim below on this topic. I think this are optional.

Comment (dlorenc) [inline README.md:17]:
```suggestion
[OKRs] (Optional)
```

File: README.md (hunk 2 above)
Comment (dlorenc) [reply README.md:63]: Could we just leave this part out and cover it in the CHARTER.md?

**Comment by dlorenc** [comment]
Anything I can do to help get this merged? I only had the two small nits left.

**Pull request merged**